geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/scalding/AccumuloSourceOptions.scala

```diff
@@ -26,21 +26,13 @@ import org.apache.log4j.Level
 import scala.util.parsing.combinator.JavaTokenParsers
 
 sealed trait AccumuloSourceOptions {
-  def instance: String
-  def zooKeepers: String
-  def user: String
-  def password: String
-  def table: String
+  def dsParams: Map[String, String]
   def logLevel: Option[Level]
-  override def toString = s"${getClass.getSimpleName}[$instance,$table]"
+  override def toString = s"${getClass.getSimpleName}[${dsParams.getOrElse("instanceId", "None")},${dsParams.getOrElse("tableName", "None")}]"
```

Comment:
i would roll back these changes and keep things as a simple case class, instead of a wrapper for a map - that way we get easy access to each param, instead of all hte 'getOrElse' with strings. (also, the accumulo formats shouldn't be tied to geomesa at all). To get the same functionality, add an object for each case class with an apply method that takes a Map and pulls out the appropriate values.